app/frontend/src/components/designer/FormsTable.vue

```diff
@@ -126,14 +126,17 @@ export default {
   name: 'FormsTable',
   data() {
     return {
+      // Assigning width: '1%' to dynamically assign width to the Table's Columns as described by this post on Stack Overflow:
+      // https://stackoverflow.com/questions/51527489/change-the-default-width-of-a-vuetifyjs-datatable-cell
       headers: [
-        { text: 'Form Title', align: 'start', value: 'name' },
+        { text: 'Form Title', align: 'start', value: 'name', width: '1%',},
```

Comment:
Missing a space before close bracket.
Also as a power-user tip, you can take advantage of shorter SO links by clicking the appropriate share button. This gives you a shorter link: https://stackoverflow.com/q/51527489

Reply:
oh, this is a nice trick ill make sure to use this from now on.

Fixed in in commit, and Rebasing.